meta/main.yml

```diff
@@ -9,13 +9,16 @@ dependencies:
       - build
 
   - role: sansible.users_and_groups
-    version: v1.0
+    version: v1.4.1
```

Comment:
This should be v1.4, this may seem a bit strange but our minor versions are canonicals for the latest patch version.